configure.ac

```diff
@@ -996,14 +996,14 @@ if test "x${enable_asan}" = "xyes"; then
   if test "x${enable_tsan}" = "xyes" -o "x${enable_tsan}" = "xstatic"; then
     AC_MSG_ERROR(Cannot have ASAN and TSAN options at the same time, pick one)
   fi
-  TS_ADDTO(AM_CFLAGS, [-fno-omit-frame-pointer -fsanitize=address])
-  TS_ADDTO(AM_CXXFLAGS, [-fno-omit-frame-pointer -fsanitize=address])
+  TS_ADDTO(AM_CFLAGS, [-fno-omit-frame-pointer -fsanitize=address -DIS_ASAN_BUILD])
```

Comment:
I originally wanted to add an "is ASAN build" flag to ink_config.h, but I couldn't figure out how to do it.